std_msgs/Header header
string label
uint32 id
geometry_msgs/Point point
BboxInfo bbox
float32 depth
